ACE/bin/make_release.py

```diff
@@ -584,30 +608,14 @@ def push ():
             vprint ("Pushing tag %s on MPC" % (tagname))
             ex ("cd $DOC_ROOT/MPC && git push origin tag " + tagname)
 
-            # Update latest tag
-            if opts.release_type == "major":
-                push_latest_tag ("ACE_TAO", "Major", tagname)
-                push_latest_tag ("ACE_TAO", "Minor", tagname)
-                push_latest_tag ("ACE_TAO", "Beta", tagname)
-                push_latest_tag ("ACE_TAO", "Micro", tagname)
-                push_latest_tag ("MPC", "ACETAO_Major", tagname)
-                push_latest_tag ("MPC", "ACETAO_Minor", tagname)
-                push_latest_tag ("MPC", "ACETAO_Micro", tagname)
-            elif opts.release_type == "minor":
-                push_latest_tag ("ACE_TAO", "Minor", tagname)
-                push_latest_tag ("ACE_TAO", "Beta", tagname)
-                push_latest_tag ("ACE_TAO", "Micro", tagname)
-                push_latest_tag ("MPC", "ACETAO_Minor", tagname)
-                push_latest_tag ("MPC", "ACETAO_Micro", tagname)
-            elif opts.release_type == "micro":
-                push_latest_tag ("ACE_TAO", "Beta", tagname)
-                push_latest_tag ("ACE_TAO", "Micro", tagname)
-                push_latest_tag ("MPC", "ACETAO_Micro", tagname)
+            # Push release branches
+            latest_branch_helper (push_latest_branch, opts.release_type)
```

Comment:
At the moment we release a major also the minor/micro branches needs to be updated, similar for a new minor, that is also a new micro

Reply:
Yes, that's what the [`latest_branch_helper` function](https://github.com/DOCGroup/ACE_TAO/pull/1182/files#diff-b13c989b2269a0d6d0f1bc09744a6f70R550) is for:

```python
>>> def latest_branch_helper (fn, release_type):
...     release_types = ("major", "minor", "micro")
...     do = release_types[release_types.index(release_type):]
...     if "micro" in do:
...         fn ("ACE_TAO", "Beta")
...         fn ("ACE_TAO", "Micro")
...         fn ("MPC", "ACETAO_Micro")
...     if "minor" in do:
...         fn ("ACE_TAO", "Minor")
...         fn ("MPC", "ACETAO_Minor")
...     if "major" in do:
...         fn ("ACE_TAO", "Major")
...         fn ("MPC", "ACETAO_Major")
... 
>>> latest_branch_helper(print, "micro")
ACE_TAO Beta
ACE_TAO Micro
MPC ACETAO_Micro
>>> latest_branch_helper(print, "minor")
ACE_TAO Beta
ACE_TAO Micro
MPC ACETAO_Micro
ACE_TAO Minor
MPC ACETAO_Minor
>>> latest_branch_helper(print, "major")
ACE_TAO Beta
ACE_TAO Micro
MPC ACETAO_Micro
ACE_TAO Minor
MPC ACETAO_Minor
ACE_TAO Major
MPC ACETAO_Major

```